modules/products/bitbucket/variables.tf

```diff
@@ -81,25 +81,25 @@ variable "admin_configuration" {
 variable "nfs_requests_cpu" {
   description = "The minimum CPU compute to request for the NFS instance"
   type        = string
-  default     = "0.25"
+  default     = "1"
```

Comment:
Are we updating default value?

Reply:
yup, it was initially updated in the [Bitbucket PR](https://github.com/atlassian-labs/data-center-terraform/pull/202/files#diff-ad9808832da6acb0c1fa14759f2b1e7bf44f157d59b0e17853a7941fb49a7efcR21) at NFS module level. Without propagating them to the outer modules, it wont take effect by default.